### PR TITLE
Feature/show map

### PR DESCRIPTION
**Pull request by Hazmid**

### CODE REVIEW

File: here_for_change/municipalities/templates/municipalities/ward_detail.html [hunk 1]
```diff
@@ -63,7 +63,7 @@ <h1 class="fvs-250 benfritz w-full text-left text-[48px] leading-[1.2] md:w-auto
   </div>
 </section>
 <section>
-  <div id="map"></div>
+    <iframe class="data-src-wardid data-src-municipalityid" src="{% url 'map' %}" frameborder="0"></iframe>
```
Comment (paulmwatson) [inline here_for_change/municipalities/templates/municipalities/ward_detail.html]: @Hazmid do we have to iframe it? Can't we embed it with JS?

File: here_for_change/static/js/map.js [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+  var zoom = 8;
+  var areas = [];
+  var youAreHereLatlng = [];
+
+  let muniAreaData = [];
+  let path = window.document.referrer.split("/municipalities")[1];
+  var municipalityId = path.split("/")[1];
+  var wardId = path.split("/")[3];
+
+  fetch(`${baseUrl}/municipalities/${municipalityId}/wards/${wardId}.json`)
```
Comment (paulmwatson) [inline here_for_change/static/js/map.js:17]: @Hazmid the other branch has the geojson data loaded into the HTML of the page, but in this branch we're calling the JSON URL for what seems to be similar or the same data?

**Review by paulmwatson — CHANGES_REQUESTED** [review]
The images in here_for_change/static/assets/ e.g. anc.png are fine for now but we'll need to make it so we can upload images in Admin as part of the data system. Images change, new party logos, new people photos etc. so we can't have it as part of the codebase in the long term.

File: here_for_change/static/js/map.js (hunk 2 above)
Comment (Hazmid) [reply here_for_change/static/js/map.js:17]: @paulmwatson  The problem was that I couldn't use the html geojson data to work in the javascript file. So getting the data in json with fetch was the work-around.

File: here_for_change/municipalities/templates/municipalities/ward_detail.html (hunk 1 above)
Comment (Hazmid) [reply here_for_change/municipalities/templates/municipalities/ward_detail.html]: @paulmwatson Iframe has been removed. map data is now embedded directly.

File: here_for_change/static/js/map.js (hunk 2 above)
Comment (paulmwatson) [reply here_for_change/static/js/map.js:17]: @Hazmid ok, so long as we're not embedding and fetching. Choose one method to get map data.

**Pull request closed without merging**